.github/workflows/ci.yml

```diff
@@ -344,19 +348,30 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build and release Docker image
         uses: docker/build-push-action@v6
+        id: build-push
         with:
           context: .
           push: ${{ startsWith(github.ref, 'refs/tags/jq-') }}
           provenance: false
           platforms: linux/386,linux/amd64,linux/arm64,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x
           tags: ${{ steps.metadata.outputs.tags }}
           labels: ${{ steps.metadata.outputs.labels }}
+      - name: Docker attest-build-provenance
+        if: startsWith(github.ref, 'refs/tags/')
+        uses: actions/attest-build-provenance@v2
+        with:
+          subject-name: ghcr.io/${{ github.repository }}
+          subject-digest: ${{ steps.build-push.outputs.digest }}
+          push-to-registry: true
 
   release:
     runs-on: ubuntu-latest
     permissions:
       contents: write
+      id-token: write
+      attestations: write
       pull-requests: write
+      actions: write
```

Comment:
Why `actions: write` is needed?

Reply:
Because of this error in the fork, which has Github defaults that are possibly different (less permissive?) than the repo settings here.

https://github.com/testimus-maximus/jq/actions/runs/12522038351/job/34929986320#step:7:51

Reply:
Okay, so drop this permission when merging right?

Reply:
You can, it just means that the repo settings here are more relaxed and the setting is implied, but on a new fork, it won't work as Github defaults are different to the repo/action settings of this project,

Reply:
It's consistently working though, so that's a bonus. Still no idea why the original fork is broken.

fake tag jq-1.8.5 https://github.com/testimus-maximus/jq/actions/runs/12522341318

https://github.com/testimus-maximus/jq/releases/tag/jq-1.8.5

Reply:
It fails because you have changed the actions workflow file. It is not needed because normally this step just commit the signature files and does not change the workflow files.

Reply:
ok, it's removed.